greenboot.spec

```diff
@@ -178,10 +206,11 @@ mkdir -p %{buildroot}%{_libexecdir}
 mkdir -p %{buildroot}%{_libexecdir}/%{name}
 mv %{buildroot}%{_bindir}/greenboot %{buildroot}%{_libexecdir}/%{name}/%{name}
 install -Dpm0644 -t %{buildroot}%{_unitdir} \
-  dist/systemd/system/*.service
+  usr/lib/systemd/system/*.service
```

Comment:
I believe this is the location in the source tarfile hence dist would be correct here? If it's the destination it should be using the correct rpm macros.

Reply:
this is in reference with [comment](https://github.com/fedora-iot/greenboot/pull/100#issuecomment-1747216121) so kept things in sync with main repo.

Reply:
for backward compatibility we can keep this as it is now.